doc/config_options.txt

```diff
@@ -1832,3 +1832,848 @@ This value is required by some providers.
+```{config:option} ceph.cluster_name storage-ceph-pool-conf
+:defaultdesc: "`ceph`"
+:shortdesc: "Name of the Ceph cluster in which to create new storage pools"
+:type: "string"
+
+```
+
+```{config:option} ceph.osd.data_pool_name storage-ceph-pool-conf
+:shortdesc: "Name of the OSD data pool"
+:type: "string"
+
+```
+
+```{config:option} ceph.osd.pg_num storage-ceph-pool-conf
+:defaultdesc: "`32`"
+:shortdesc: "Number of placement groups for the OSD storage pool"
+:type: "string"
+
+```
+
+```{config:option} ceph.osd.pool_name storage-ceph-pool-conf
+:defaultdesc: "name of the pool"
+:shortdesc: "Name of the OSD storage pool"
+:type: "string"
+
+```
+
+```{config:option} ceph.rbd.clone_copy storage-ceph-pool-conf
+:defaultdesc: "`true`"
+:shortdesc: "Whether to use RBD lightweight clones"
+:type: "bool"
+Enable this option to use RBD lightweight clones rather than full dataset copies.
+```
+
+```{config:option} ceph.rbd.du storage-ceph-pool-conf
+:defaultdesc: "`true`"
+:shortdesc: "Whether to use RBD `du`"
+:type: "bool"
+This option specifies whether to use RBD `du` to obtain disk usage data for stopped instances.
+```
+
+```{config:option} ceph.rbd.features storage-ceph-pool-conf
+:defaultdesc: "`layering`"
+:shortdesc: "Comma-separated list of RBD features to enable on the volumes"
+:type: "string"
+
+```
+
+```{config:option} ceph.user.name storage-ceph-pool-conf
+:defaultdesc: "`admin`"
+:shortdesc: "The Ceph user to use when creating storage pools and volumes"
+:type: "string"
+
+```
+
+```{config:option} source storage-ceph-pool-conf
+:shortdesc: "Existing OSD storage pool to use"
+:type: "string"
+
+```
+
+```{config:option} volatile.pool.pristine storage-ceph-pool-conf
+:defaultdesc: "`true`"
+:shortdesc: "Whether the pool was empty on creation time"
+:type: "string"
+
+```
+
+<!-- config group storage-ceph-pool-conf end -->
+<!-- config group storage-ceph-volume-conf start -->
+```{config:option} block.filesystem storage-ceph-volume-conf
+:condition: "block-based volume with content type `filesystem`"
+:defaultdesc: "same as `volume.block.filesystem`"
+:shortdesc: "File system of the storage volume"
+:type: "string"
+Valid options are: `btrfs`, `ext4`, `xfs`
+If not set, `ext4` is assumed.
+```
+
+```{config:option} block.mount_options storage-ceph-volume-conf
+:condition: "block-based volume with content type `filesystem`"
+:defaultdesc: "same as `volume.block.mount_options`"
+:shortdesc: "Mount options for block-backed file system volumes"
+:type: "string"
+
+```
+
+```{config:option} lvm.stripes storage-ceph-volume-conf
+:defaultdesc: "same as `volume.lvm.stripes`"
+:shortdesc: "Number of stripes to use for new volumes (or thin pool volume)"
+:type: "string"
+
+```
+
+```{config:option} lvm.stripes.size storage-ceph-volume-conf
+:defaultdesc: "same as `volume.lvm.stripes.size`"
+:shortdesc: "Size of stripes to use"
+:type: "string"
+The size must be at least 4096 bytes, and a multiple of 512 bytes.
+```
```

Comment:
Did you regenerate the file? These options should not be there for Ceph.

Reply:
Ah yes good catch! I'm fixing it

Reply:
@ru-fu should be ok now